plugins/inputs/system/linux_sysctl_fs.go

```diff
@@ -80,9 +89,10 @@ func (sfs *SysctlFS) Gather(acc telegraf.Accumulator) error {
 }
 
 func init() {
+
 	inputs.Add("linux_sysctl_fs", func() telegraf.Input {
 		return &SysctlFS{
-			path: "/proc/sys/fs",
+			path: path.Join(GetHostSys(), "/fs"),
```

Comment:
I guess this should actually use GetHostProc too, since `/proc/sys` and `/sys` are not the same.